examples/bno055_webgl_demo/static/css/bootstrap.css.license

```diff
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```

Comment:
bootstrap.css is still listed as copyright 2021 ladyada here.

The bootstrap license page shows copyright twitter 2018 https://getbootstrap.com/docs/4.0/about/license/

Reply:
latest commit fixes all of the remaining bootstrap licenses that I found.